fortinet-fortiweb-cloud/operations.py

```diff
@@ -0,0 +1,166 @@
+import requests, json
+import urllib.parse
+from .constants import *
+from connectors.core.connector import get_logger, ConnectorError
+
+logger = get_logger('fortinet-fortiweb-cloud')
+
+
+class FortiWeb(object):
+    def __init__(self, config, *args, **kwargs):
+        self.api_key = config.get('api_key')
+        url = config.get('server_url').strip('/')
+        if not url.startswith('https://') and not url.startswith('http://'):
+            self.url = 'https://{0}/v2'.format(url)
+        else:
+            self.url = url + '/v2'
+        self.verify_ssl = config.get('verify_ssl')
+
+    def make_rest_call(self, url, method, data=None, params=None):
```

Comment:
Can we provide a default value for the argument method
method='GET'
As most of the action are of GET type